### PR TITLE
fix(spelling): run most code through a spell checker

### PR DESCRIPTION
**Pull request by Haroenv**
I did change `unserialize` to `deserialize`, so it's not only comments and text that changed, but it's not a public method, so it should be okay.

### CODE REVIEW

File: src/store.js [hunk 1]
```diff
@@ -199,6 +199,7 @@ export class Store {
     return this._helper.lastResults.processingTimeMS;
   }
 
+  // todo: change to goToFirstPage()
```
Comment (Haroenv) [inline src/store.js]: I'm not sure how to change this one without breaking change

Comment (rayrutjes) [inline src/store.js]: I was thinking of removing those.
We don't use them.

Comment (rayrutjes) [inline src/store.js]: BTW that is what Beta's are for, I doubt there are custom components relying on that un-documented method.

Comment (rayrutjes) [inline src/store.js]: Please do a separate PR though if you are to merge this one.

Comment (Haroenv) [inline src/store.js]: Will do a second PR to remove this

**Comment by rayrutjes** [comment]
You push it, you merge it @Haroenv  😁
